tests/tools/TestApp/src/mcm.c

```diff
@@ -49,16 +51,19 @@ int mcm_send_video_frames(MeshConnection *connection, const char *filename, int
         }
         read_size = fread(buf->payload_ptr, 1, buf->payload_len, file);
         if (read_size == 0) {
+            mesh_buffer_set_payload_len(buf, 0);
+            mesh_put_buffer(&buf);
```

Comment:
`mesh_put_buffer(&buf)` can return error, you should check it

Reply:
done